test/controllers/microposts_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
```

Comment:
you don't have to write test code.
The grammar of the test code of our system is a little bit different from this one.
we use rspec. this is minitest.